src/resources/views/crud/fields/icon_picker.blade.php

```diff
@@ -72,33 +72,55 @@
     {{-- FIELD EXTRA CSS  --}}
     @push('crud_fields_styles')
         {{-- The chosen font --}}
-        <link rel="stylesheet" type="text/css" href="{{ $field['font_icon_file_path'] }}">
+            <link rel="stylesheet" type="text/css" href="{{ $field['font_icon_file_path'] }}">
         <!-- Bootstrap-Iconpicker -->
         <link rel="stylesheet" href="{{ asset('packages/bootstrap-iconpicker/bootstrap-iconpicker/css/bootstrap-iconpicker.min.css') }}"/>
     @endpush
 
     {{-- FIELD EXTRA JS --}}
     @push('crud_fields_scripts')
         <!-- Bootstrap-Iconpicker -->
-        <script type="text/javascript" src="{{ asset('packages/bootstrap-iconpicker/bootstrap-iconpicker/js/bootstrap-iconpicker.bundle.min.js') }}"></script>
 
+        <script type="text/javascript" src="{{ asset('packages/bootstrap-iconpicker/bootstrap-iconpicker/js/bootstrap-iconpicker.bundle.min.js') }}"></script>
+        @if($field['iconset'] == 'lineawesome')
+            <script type="text/javascript" src="{{ asset('packages/backpack/crud/js/fields/iconpicker/line-icons.js') }}"></script>
+        @endif
         {{-- Bootstrap-Iconpicker - set hidden input value --}}
         <script>
             function bpFieldInitIconPickerElement(element) {
                 var $iconset = element.attr('data-iconset');
                 var $iconButton = element.siblings('button[role=icon-selector]');
                 var $icon = element.attr('value');
+                var $customIconSet = null;
+
+                // if the iconset is lineawesome we create our own iconset for the iconpicker.
+                if($iconset == 'lineawesome') {
+                    // we store it in global window object to avoid fetching the same information again
+                    // in case we have two or more iconpickers with lineawesome
+                    if(typeof window.icon_pickerLineIcons !== "undefined") {
```

Comment:
Any chance of not having camelCase combined with snake_case? 😅
`icon_pickerLineIcons` > `iconPickerLineIcons`?

Also, maybe that variable should be an array, being keys the name of the set, `window.iconPickerIcons['lineawesome']`.
That way developers can add their own.

Reply:
hehe sure @promatik 

this is like POC code, I don't expect this to be merged before 4.2, or to be merged at all 🗡️ 

we will deal with it on time if we have to 👍